auditwheel 5.1.0: policy.json not installed when dist-info created first / when running from git tarball

When I run the tests over installed auditwheel 4.0.0 from source, It fails with:

```
==================================== ERRORS ====================================
___________________ ERROR collecting tests/unit/test_main.py ___________________
tests/unit/test_main.py:4: in <module>
    from auditwheel.main import main
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/main.py:11: in <module>
    from . import main_repair
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/main_repair.py:4: in <module>
    from .policy import (load_policies, get_policy_by_name, get_policy_name,
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/policy/__init__.py:59: in <module>
    with open(join(dirname(abspath(__file__)), 'policy.json')) as f:
E   FileNotFoundError: [Errno 2] No such file or directory: '/builddir/build/BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/policy/policy.json'
__________________ ERROR collecting tests/unit/test_policy.py __________________
tests/unit/test_policy.py:5: in <module>
    from auditwheel.policy import get_arch_name, get_policy_name, \
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/policy/__init__.py:59: in <module>
    with open(join(dirname(abspath(__file__)), 'policy.json')) as f:
E   FileNotFoundError: [Errno 2] No such file or directory: '/builddir/build/BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/policy/policy.json'
__________________ ERROR collecting tests/unit/test_repair.py __________________
tests/unit/test_repair.py:5: in <module>
    from auditwheel.repair import append_rpath_within_wheel
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/repair.py:17: in <module>
    from .policy import get_replace_platforms
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/policy/__init__.py:59: in <module>
    with open(join(dirname(abspath(__file__)), 'policy.json')) as f:
E   FileNotFoundError: [Errno 2] No such file or directory: '/builddir/build/BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/policy/policy.json'
________________ ERROR collecting tests/unit/test_wheel_abi.py _________________
tests/unit/test_wheel_abi.py:5: in <module>
    from auditwheel import wheel_abi
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/wheel_abi.py:17: in <module>
    from .policy import (lddtree_external_references, versioned_symbols_policy,
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/policy/__init__.py:59: in <module>
    with open(join(dirname(abspath(__file__)), 'policy.json')) as f:
E   FileNotFoundError: [Errno 2] No such file or directory: '/builddir/build/BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/policy/policy.json'
=============================== warnings summary ===============================
../../BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/patcher.py:2
  /builddir/build/BUILDROOT/auditwheel-4.0.0-1.fc35.x86_64/usr/lib/python3.10/site-packages/auditwheel/patcher.py:2: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
    from distutils.spawn import find_executable

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================== short test summary info ============================
ERROR tests/unit/test_main.py - FileNotFoundError: [Errno 2] No such file or ...
ERROR tests/unit/test_policy.py - FileNotFoundError: [Errno 2] No such file o...
ERROR tests/unit/test_repair.py - FileNotFoundError: [Errno 2] No such file o...
ERROR tests/unit/test_wheel_abi.py - FileNotFoundError: [Errno 2] No such fil...
!!!!!!!!!!!!!!!!!!! Interrupted: 4 errors during collection !!!!!!!!!!!!!!!!!!!!
========================= 1 warning, 4 errors in 0.33s =========================
```

The policy.json file does not appear to be installed.


I was wondering why the official wheel has it.


My observations so far are:

 - when I build the wheel from sdist directly, policy.json is installed
 - when I run `prepare_metadata_for_build_wheel` hook first, it deletes the egg-info a creates dist-info, policy.json is not installed
 - when I build the wheel from git directly, policy.json is installed
 - when I build the wheel from [git archive](https://github.com/pypa/auditwheel/archive/4.0.0/auditwheel-4.0.0.tar.gz) with `export PBR_VERSION=4.0.0`, policy.json is not installed

Is there a way to make policy.json always installed? Should it be listed somewhere explicitly as a data file?

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,8 +1,12 @@
 include README.rst
 include LICENSE
+include CHANGELOG.md
+include src/auditwheel/policy/*.json
+include src/auditwheel/_vendor/wheel/LICENSE.txt
 
 exclude .coveragerc
 exclude .gitignore
+exclude .git-blame-ignore-revs
 exclude .pre-commit-config.yaml
 exclude .travis.yml
 exclude noxfile.py

